### ISSUE TITLE
Improve Documentation

### ISSUE THREAD

**Issue by odalysadam** [twg/devour#212]
So just a friendly hint and maybe I'm not having the brightest day, but without much prior knowledge to the topic of json:apis I found your documention rather hard. 
You get very much into the Initializer and the advanced features, but skip the basic parts like the Model defining process and how the attributes get mapped. (Having a "post" as a type here is just a tad misleading btw.) 
If you could answer simple questions in your documentation like - "What are the parameters for those functions define, create etc.?", "Is it possible to define multiple models?", "Does update use PUT or PATCH?", "Can you define a Model as Read Only?" and I'm sure there's more - out of the box, you would give people a much easier start into using your project instead of looking elsewhere. Convenience is key. Make your project great again.

Much love

**Comment by tijn** [comment]
I agree @odalysadam , it could be improved a lot. Would you perhaps create a pull request with what you would like to see?
